server/views/review/finalChecks.pug

```diff
@@ -35,4 +35,7 @@ block content
             span.block Comments
             span#comments.block.bold
               | #{finalChecks.confiscationOrder.comments}
-
+    
```

Comment:
while I don't doubt this is correct, I can't see where this page is used in the journey. Is it an orphaned page that is no longer used?

Reply:
It's pulling through final check info to be visible to the DM when they view this section from the tasklist before making their final decision. I've added a screen shot of the page content for this above.